telemetry/specs/INT.mdk

```diff
@@ -342,9 +346,12 @@ This may be addressed in the following ways -
 * It is recommended that the MTU of links between INT sources and sinks is
 configured to be higher than the MTU of preceding link MTUs (server/VM NIC MTUs)
 by an appropriate amount. Configuring an MTU differential of
-(Per-hop Metadata Length\*4\*INT Hop Count + 8), based on conservative values of
+[Per-hop Metadata Length\*4\*INT Hop Count + 12] bytes, based on conservative values of
```

Comment:
Thanks for catching the error. Thinking more, even Metadata + 12 bytes is not enough. We have to look at the maximum overhead added by INT across different encapsulation types. For TCP/IP, the overhead is Per-hop ML*4*Number of Hops + 16 (4 bytes shim header + 8 bytes INT header + 4 bytes Tail header).

Reply:
Done. I called out the total Fixed INT Header Length as another variable and defined it per encap types.